--- ../../../../modules/ext.math.mathjax.enabler.js
+++ ../../../modules/ext.math.mathjax.enabler.js
@@ -14,7 +14,7 @@
   mathJax.Config = function () {
     MathJax.Hub.Config({
       root: mw.config.get('wgExtensionAssetsPath') + '/Math/modules/MathJax',
-      config: ['TeX-AMS-texvc_HTML.js'],
+      config: ['TeX-AMS-texvc_HTML.js', 'local/local.js'],
       'v1.0-compatible': false,
       styles: {
         '.mtext': {

